clients/fides-js/src/lib/tcf/types.ts

```diff
@@ -5,6 +5,16 @@ import type {
   UserConsentPreference,
 } from "../consent-types";
 
+export enum LegalBasisForProcessingEnum {
```

Comment:
these are mostly all copied from the updated admin-ui types. I rearranged them a bit in the file to make more intuitive sense (hopefully)